File: lib/point.ts
export default class Point {
  x: number;
  y: number;

  constructor(x: number, y: number) {
    this.x = x;
    this.y = y;
  }

  // Returns the distance between two points
  getDistance(point: Point) {
    return Math.sqrt(
      Math.pow(this.x - point.x, 2) + Math.pow(this.y - point.y, 2)
    );
  }

  toString() {
    return `(${this.x}, ${this.y})`;
  }
}

File: py/lib/point.py
class Point:
    def __init__(self, x: float, y: float):
        self.x = x
        self.y = y

    # Returns the distance between two points
    def get_distance(self, point):
        return ((self.x - point.x) * 2 + (self.y - point.y) * 2) ** 0.5

    def __str__(self):
        return f"({self.x}, {self.y})"